lib/export/coldp/files/type_material.rb

```diff
@@ -0,0 +1,127 @@
+# ID
+# nameID
+# citation
+# status
+# referenceID
+# locality
+# country
+# latitude
+# longitude
+# altitude
+# host
+# date
+# collector
+# institutionCode
+# catalogNumber
+# associatedSequences
+# sex
+# link
+# remarks
+#
+module Export::Coldp::Files::TypeMaterial
+
+  def self.locality(co)
+    [co.dwc_country, co.dwc_state_province, co.dwc_county, co.dwc_locality].compact.join(', ')
+  end
+
+  # TODO: This very likely includes more than just hosts (symbiotypes):
+  #   could possibly be narrowed down by checking if the biological relationship type's inverted name includes the word host?
+  #     we are specifying the relationship type in the string to make it clearer
+  def self.host(o, co)
+    unless co.biological_associations.any?
+      return nil
+    end
+    hosts = []
+    co.biological_associations.where(biological_association_object_type: 'Otu').each do |ba|
+      if o.taxon_name_id.nil?
+        object_name = o.name
+      else
+        tn = TaxonName.find(o.taxon_name_id)
+        object_name = tn.cached
+      end
+      if ba.biological_relationship.inverted_name.include? 'host'
```

Comment:
@gdower I think this wont' really work, or catch only very few posibilities.  I'd suggest leaving out host for now?